client/sites/settings/controller.tsx

```diff
@@ -227,5 +230,13 @@ export function dashboardBackportSiteSettings( context: PageJSContext, next: ()
 		/>
 	);
 
+	await Promise.all( [
```

Comment:
Honestly, I'm not seeing a noticeable performance boost when testing on Atomic site 🤔 It's still slow; we're still blocking the view until we retrieve the response from the site endpoint 🤔 

I think the a more proper way is to preload the router when we hover on the v1 Settings tab. Have we considered something like that? I see a promising thing here:

https://github.com/Automattic/wp-calypso/blob/af8cf1bf7579259c0deb1a2640f38f1dc6e5ce53/client/components/section-nav/item.jsx#L61